tf-ansible/aws.yaml

```diff
@@ -0,0 +1,119 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+
+imports:
+  - https://cloudify.co/spec/cloudify/6.2.0/types.yaml
+  - plugin:cloudify-terraform-plugin
+  - plugin:cloudify-ansible-plugin
+
+inputs:
+  aws_region_name:
+    type: string
+    description: AWS Region to deploy resources into
+    default: us-west-2
```

Comment:
Why are we defaulting to `us-west-2`. In the past, Nati asked that we default our examples to us-east-1. Also if there is a range of options in the terraform zip, we can constrain these, see [here](https://github.com/cloudify-community/blueprint-examples/blob/master/virtual-machine/aws-terraform.yaml#L22).